Task1 5

lizsharp opened the pull request.

lizsharp committed:
Added gitignore

lizsharp committed:
Task 1-5

Research
Design
Scaffolding
About
Portfolio

sjr-mayden commented on index.html:
```diff
@@ -0,0 +1,142 @@
+    <title>Liz Sharp Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="style.css">
+
```
Extra line here, can remove

sjr-mayden commented on index.html:
```diff
@@ -0,0 +1,142 @@
+
+</head>
+<body>
+
```
Extra line here, can remove

sjr-mayden commented on index.html:
```diff
@@ -0,0 +1,142 @@
+            <img class="hero-image" src="assets/beautiful-blur-bright-326055.2.png" alt="butterfly">
+            <div class="name">Liz Sharp</div>
+        </div>
+
```
Extra line here, can remove

sjr-mayden commented on index.html:
```diff
@@ -0,0 +1,142 @@
+        </div>
+
+        <nav>
+
```
Extra line here, can remove

sjr-mayden commented on index.html:
```diff
@@ -0,0 +1,142 @@
+            </div>
+        </nav>
+
+        <! portfolio section >
```
HTML comments should be:

<!-- portfolio section -->

sjr-mayden commented on index.html:
```diff
@@ -0,0 +1,142 @@
+                </div>
+
+                <div class="portfolio-row">
+                    <div class="portfolio-item">
```
Is there going to be a link here? Currently no a tag

sjr-mayden commented on style.css:
```diff
@@ -0,0 +1,210 @@
+    padding: 0;
+}
+
+body{
```
No space between end of selector and {, should be one

sjr-mayden commented on style.css:
```diff
@@ -0,0 +1,210 @@
+body{
+    font-size: 62.5%;
+}
+
```
One line between CSS groups please

sjr-mayden commented on style.css:
```diff
@@ -0,0 +1,210 @@
+
+.background-container{
+    float: left;
+    clear:both;
```
Space between : and the value

sjr-mayden commented:
Looks like extra images are included, could be trimmed and the files renamed

sjr-mayden commented on index.html:
```diff
@@ -0,0 +1,142 @@
+
+                <div class="background-container">
+                    <h3 class="bio-heading">Background</h3>
+                    <p class="bio-text">Hereereereere Hereereereere is some text about the portfolio project. Here is so is some text about tis some text about the portfolio project. Here is so is some text about the portfolio project. Here is some text about the is some text about the portfolio project. Here is some text about the portfolio project. Here is some text about the portfolio project. ...
```
Spelling mistake :)

sjr-mayden commented on index.html:
```diff
@@ -0,0 +1,142 @@
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,intital-scale=1">
```
*initial

lizsharp committed:
Made the following changes:

Got rid of extra lines in html
Corrected html comment tag
Added link code to portfolio images
Removed spaces between selector and curly bracket
Removed extra lines between css
Added space between property and value in css
Corrected spelling mistake in bio text, replaced it with lorem ipsum text instead
Corrected spelling mistake line 6 'initial'

lizsharp committed:
Changes made:

Removed extra lines at end of css
Renamed images in asset directory and in corresponding html
